plenum/test/replica/test_replica_checkpoint_validation.py

```diff
@@ -1,127 +0,0 @@
-import pytest
-
-from plenum.common.messages.node_messages import Checkpoint
-from plenum.common.startable import Mode
-from plenum.server.replica_validator import ReplicaValidator
-from plenum.server.replica_validator_enums import DISCARD, INCORRECT_INSTANCE, PROCESS, CATCHING_UP, ALREADY_STABLE, \
-    STASH_CATCH_UP, OLD_VIEW, FUTURE_VIEW, STASH_VIEW
-
-
```

Comment:
Did we replace this test by a new one?

Reply:
Yes, we change it to /test/consensus/checkpoint_service/test_checkpoint_validation.py